### PR TITLE
update for rubocop 0.49

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -1,56 +1,35 @@
...
 # `rubocop --auto-gen-config`
-# on 2016-08-24 16:11:34 -0700 using RuboCop version 0.42.0.
+# on 2017-09-06 09:08:28 -0700 using RuboCop version 0.49.1.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Manually added to prevent `unrecognized cop` warnings during execution.
-require: rubocop-rspec
```
Comment (jcoyne) [inline .rubocop_todo.yml]: Do we need to add this back in?

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -62,18 +41,11 @@ RSpec/NamedSubject:
...
   Exclude:
-    - 'spec/**/*_spec.rb'
-
-# Offense count: 2
-# Configuration parameters: EnforcedStyle, SupportedStyles.
-# SupportedStyles: not_to, to_not
-RSpec/NotToNot:
-  Exclude:
-    - 'spec/unit/catalog_spec.rb'
+    - 'spec/unit/blacklight/access_controls/search_builder_spec.rb'
```
Comment (jcoyne) [inline .rubocop_todo.yml]: I think we should continue to exclude everything in spec from this.  Maybe it needs to move to .rubocop.yml?

File: .rubocop_todo.yml (hunk 1 above)
Comment (dunn) [reply .rubocop_todo.yml]: I'll add it to rubocop.yml

Comment (dunn) [reply .rubocop_todo.yml]: Oh, it's already there.

**Pull request merged**